src/main/java/bookstore/model/Book.java

```diff
@@ -0,0 +1,22 @@
+package bookstore.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Entity
+@Data
```

Comment:
```suggestion
```

Don't use `@Data` for entities, it can cause problems
Lombok has other annotations to avoid boilerplate code